server/modules/search/elasticsearch/engine.js

```diff
@@ -92,30 +93,16 @@ module.exports = {
         index: this.config.indexName,
         body: {
           query: {
-            bool: {
-              filter: [
-                {
-                  bool: {
-                    should: [
-                      {
-                        simple_query_string: {
-                          query: q
-                        }
-                      },
-                      {
-                        query_string: {
-                          query: `*${q}*`
-                        }
-                      }
-                    ],
-                    minimum_should_match: 1
-                  }
-                }
-              ]
+            simple_query_string: {
+              query: `*${q}*`,
+              fields: ['title^20', 'description^3', 'tags^8', 'content^1'],
+              default_operator: 'and',
+              analyze_wildcard: true
             }
           },
           from: 0,
           size: 50,
+          explain: true,
```

Comment:
Necessary in non-dev?

Reply:
It is indeed not necessary i will omit this line